packages/strapi-plugin-graphql/services/Aggregator.js

```diff
@@ -75,22 +77,6 @@ module.exports = {
     }, {});
   },
 
-  /**
-   * Returns a list of fields that have type included in fieldTypes.
-   */
-  getFieldsByTypes: (fields, typeCheck, returnType) => {
```

Comment:
Why did you delete this function ?
It still call many time `this. getFieldsByTypes`

Reply:
because it was duplicated, there was another exactly the same

Reply:
I'm sorry it was a mistake, I've already corrected it.